share/site/duckduckgo/styleguide.tx

```diff
@@ -21,7 +21,6 @@
 				<li><a href="#txt-n-color"><: l("Text & Colors") :></a></li>
 				<li><a href="#forms"><: l("Forms") :></a></li>
 				<li><a href="#grids"><: l("Grids") :></a></li>
-				<li><a href="#popovers">Popovers</a></li>
```

Comment:
Because we don't currently have a way to do them without initializing and rendering them through handlebars, and I'm not really sure that it's worth coding in the workaround to those modal views.